### DISCUSSION TITLE
showVisitedLinks not working for some theme variants

### DISCUSSION THREAD

**Discussion by skiboyjcmu** [McShelby/hugo-theme-relearn#517] [General]
my team really likes the showVisitedLinks feature in the menu to help indicate to participants that they've completed (or at least visited) a task/chapter.  Having recently migrated from Learn to ReLearn theme, I'm finding the check marks aren't showing up in certain Themes.  You can replicate this with the [reLearn theme page](https://mcshelby.github.io/hugo-theme-relearn/index.html) and the variant switcher at the bottom of nav menu

Checkmarks show up on the Relearn Light | Brighti | Neon, but not on any of the other variants.  I haven't been able to identify why it's behaving this way.  Any ideas?

![image](https://user-images.githubusercontent.com/20427852/236003680-b51a881f-5dd2-49a6-a628-6bc3f3e93d80.png)

**Comment by McShelby** [comment]
I've created an issue for this #518

**Reply by skiboyjcmu** [reply]
sweet, thx! I'll add some detail in the issue as the behavior appears to be somewhat haphazard.  If you switch between the themes, you can sometimes see the check marks appear then go away/get painted over maybe?  And just now I'm seeing check marks appear for other chapters which have been visited, but not the currently selected one: 
![image](https://user-images.githubusercontent.com/20427852/236295317-faffc63a-802c-4275-bfaf-f89f911e43c3.png)

**Reply by McShelby** [reply]
Thanks for reporting. I've noticed this myself when I was confirming your issue report. Really strange behavior.
